pkg/loki/config_wrapper_test.go

```diff
@@ -1054,10 +1093,12 @@ func TestLoopbackAppendingToFrontendV2(t *testing.T) {
 	assert.NoError(t, err)
 	assert.NotEmpty(t, defaultIface)
 
-	t.Run("by default, loopback should be in FrontendV2 interface names", func(t *testing.T) {
+	t.Run("when using common or ingester ring configs, loopback should be added interface names", func(t *testing.T) {
 		config, _, err := configWrapperFromYAML(t, minimalConfig, []string{})
 		assert.NoError(t, err)
 		assert.Equal(t, []string{"eth0", "en0", defaultIface}, config.Frontend.FrontendV2.InfNames)
+		assert.Equal(t, []string{"eth0", "en0", defaultIface}, config.Ingester.LifecyclerConfig.InfNames)
+		assert.Equal(t, []string{defaultIface}, config.Common.Ring.InstanceInterfaceNames)
```

Comment:
This got me confused, shouldn't this scenario fail?

Reply:
our current defaults for the common ring config for interface names is an empty array, so when we append the default it's the only item in the array. I picked that up from your work so I wasn't sure what the "correct" behavior was supposed to be?

Reply:
ohhh makes sense. At first, I thought that the expected default interface list for the common ring would be `["eth0", "en0", defaultIface]`, but as you pointed out, the default value is just a []. 

Resolved!